### DISCUSSION TITLE
How can pulto  delete the  recent  my work ?

### DISCUSSION THREAD

**Discussion by liu20001028** [fonsp/Pluto.jl#3015] [Questions about using Pluto]
how can l  delete my works          althought  l delete it in my  PC but  lt alway show in Pluto

**Comment by disberd** [comment]
are you referring to the list of "recent" notebooks in the main Pluto page?
That list is stored in the browser storage (as mentioned in https://github.com/fonsp/Pluto.jl/issues/877) and you have to clear it from the browser by typing `localStorage.clear()` in the browser's developer console (see https://julialang.zulipchat.com/#narrow/stream/243342-pluto.2Ejl/topic/Remove.20links.20on.20landing.20page)

**Reply by liu20001028** [reply]
yes  l   l already did this , But  l think  it  would  be better  if  Pluto a dd a "delete" button

**Reply by liu20001028** [reply]
Thank you for your contribution to Pluto. We really love Pluto！！     I  hope  you can  make pluto  like   jupyter !
Maybe  I  think  you can   make Pluto  as  a IDE  App    with  date  science  with support   julia   Pyton   !  
 julia  not  have  a  special IDE   athough Vscode is  really good enought !
